### DISCUSSION TITLE
Extract Graph Dependencies

### DISCUSSION THREAD

**Discussion by yilinxia** [arviz-devs/arviz_dashboard#27] [General]
Hi team,

From an implementation standpoint, when our objectives are clear, building the dashboard using static visualizations through iterative exploration and implementation seems straightforward. But when we pivot to a research perspective, things get a bit more vague. We might need to delve into how we can seamlessly extract dependencies across different visualizations. Additionally, analyzing these dependencies could pave the way for a more structured and formalized approach to dashboard construction.

Would love to get your thoughts on this approach and any experiences you might want to share.

**Comment by aloctavodia** [comment]
I am not sure where these "dependencies across different visualizations" live and then not sure how to "extract" them, or from where. Are these dependencies of a statistical nature? Can/should we frame them in terms of a Bayesian Worflow. A trivial example you can not diagnose an MCMC sample unless you first sample from a model. So that establishes a "natural" dependency. Is this what you are thinking about or something else?

Can you provide an example from a different domain, where this kind of dependency analysis has already been done. Or can you provide a toy example, like I don't know a dashboard for a car or something similar?
